src/test/java/teammates/test/cases/ui/InstructorCourseEditPageActionTest.java

```diff
@@ -128,7 +130,16 @@ public void testExecuteAndPostProcess() throws Exception {
         }
 
     }
-
+    
+    public List<InstructorAttributes> getInstructorList(List<CourseEditInstructorPanel> instructorPanelList) {
+        ArrayList<InstructorAttributes> instructorList = new ArrayList<InstructorAttributes>();
```

Comment:
`List<InstructorAttributes>`.

As an alternative, you may find it tidier to simply modify `verifySameInstructorList()` to support a `List<CourseEditInstructorPanel>` passed in instead of making a method to extract a list of instructors.